packages/core/core/src/PackagerRunner.js

```diff
@@ -394,7 +395,15 @@ export default class PackagerRunner {
 
     let packager = await this.config.getPackager(bundle.name);
     let {name, resolveFrom, plugin} = packager;
+    let measurement;
     try {
+      measurement = applicationProfiler.createMeasurement(name, {
```

Comment:
Should this be gated by a check of `applicationProfiler.enabled`? Alternatively, could `applicationProfiler` expose category-specific methods that take positional args (instead of requiring object allocations) that are no-op by default?

Reply:
Yeah, good idea - given most of the calls follow the same pattern with just name, category and the filename changing can create a function for that use case with 3 string args that exits early if disabled (or as you say, change all of these to `applicationProfiler.enabled && applicationProfiler.createMeasurement(..)` - that would make `measurement` falsy which means later the `measurement && measurement.end()` would work.